pkg/gui/files_panel.go

```diff
@@ -286,6 +288,17 @@ func (gui *Gui) handleCommitPress(g *gocui.Gui, filesView *gocui.View) error {
 		return gui.createErrorPanel(gui.Tr.SLocalize("NoStagedFilesToCommit"))
 	}
 	commitMessageView := gui.getCommitMessageView()
+	prefixPattern := gui.Config.GetUserConfig().GetString("git.commitPrefixes." + utils.GetCurrentRepoName() + ".pattern")
+	prefixReplace := gui.Config.GetUserConfig().GetString("git.commitPrefixes." + utils.GetCurrentRepoName() + ".replace")
+	if len(prefixPattern) > 0 && len(prefixReplace) > 0 {
+		rgx := regexp.MustCompile(prefixPattern)
```

Comment:
given prefixPattern is provided by the user, I wouldn't use MustCompile here. I would use the alternative method that returns an error if something goes wrong, and then tell the user via `gui.createErrorPanel` that their pattern doesn't compile

Reply:
Makes sense. Fixed it.